pkg/daemon/daemon.go

```diff
@@ -299,6 +306,15 @@ func (dn *Daemon) operatorConfigChangeHandler(old, new interface{}) {
 		dn.disableDrain = newDisableDrain
 		log.Log.Info("Set Disable Drain", "value", dn.disableDrain)
 	}
+
+	if !reflect.DeepEqual(oldCfg.Spec.FeatureGates, newCfg.Spec.FeatureGates) {
+		dn.featureGate.Init(newCfg.Spec.FeatureGates)
+		log.Log.Info("Enabled featureGates", "featureGates", dn.featureGate.String())
```

Comment:
nit: as we initialize this one on the daemon creation maybe here we can say something like updated featureGates?